core/base/inc/TROOT.h

```diff
@@ -297,6 +299,7 @@ friend TROOT *ROOT::Internal::GetROOT2();
    void              SaveContext();
    void              SetApplication(TApplication *app) { fApplication = app; }
    void              SetBatch(Bool_t batch = kTRUE) { fBatch = batch; }
+   void              SetRandomAccessCompression(Bool_t random = kTRUE) { fRandomAccessCompression = random; }
```

Comment:
I agree. I actually tried to create such a bit in TTree as well as TBranch. But I saw TTree need to call several functions(such as TBranchImpl and TBranchBonc(?or something)) to create a TBranch. Do you think we also need to put a bit in TBranch?